scripts/eurostat/health_determinants/common/import_download_details.py

```diff
@@ -0,0 +1,42 @@
+download_details = {
```

Comment:
Please add a copyright header and a comment indicating the structure of this dict